libkode/printer.h

```diff
@@ -151,6 +159,16 @@ class KODE_EXPORT Printer
 
 
   private:
+    /**
+     * @brief printCodeIntoFile
+     * Writes the string passed through the code parameter to the file refernced
+     * by the file parameter in the case if it differs from the content of the file pointed by the
+     * file parameter.
+     * @param code reference to a Code object which contents needs to be printed
+     * @param file the target file in unopened state with filename set
+     */
+    void printCodeIntoFile( const Code &code, QFile *file );
```

Comment:
You could move this method to the Private class, to keep the header smaller and cleaner.

Reply:
Fixed.